client/src/components/Chat/Menus/Endpoints/MenuItem.tsx

```diff
@@ -136,12 +134,11 @@ const MenuItem: FC<MenuItemProps> = ({
               <div className="text-token-text-primary" key={`set-key-${endpoint}`}>
                 <button
                   className={cn(
-                    'invisible flex gap-x-1 group-hover:visible',
+                    ' invisible flex gap-x-1 group-hover:visible',
                     selected ? 'visible' : '',
                     expiryTime
-                      ? 'w-full rounded-lg p-2 hover:text-gray-400 dark:hover:text-gray-400'
+                      ? 'w-full rounded-lg p-2 text-gray-400 hover:text-black dark:text-gray-400 dark:hover:text-white'
```

Comment:
Any time you touch elements with legacy colors, please apply theme classnames, especially don't add legacy aka `*-gray-*`